src/App.js

```diff
@@ -233,12 +233,45 @@ class App extends Component {
       credentials: "same-origin",
       method: "POST",
       headers: {
-        // 'Authorization': 'Basic ' + btoa(getUserName() + ":" + getPassword())
+        // next line needs be commented out on prod, but active on dev
+        // 'Authorization': 'Basic ' + btoa(getUserName() + ":" + getPassword()),
         'X-CSRFToken': csrftoken
       },
       body: form
     };
-    fetch(url, opts);
+    var responseResult = {};
+    fetch(url, opts)
+      .then(response => {
+
+        responseResult = response;
+        if (response.status < 300) {
+          return response;
+        } else {
+          // The result cannot be parsed if it succeeds.
+          return response.json();
+        }
+
+      })
+      .then(result => {
+
+        var message = "";
+        if (responseResult.status == 204) {
+          message = "De bestanden zijn verstuurd.";
+        } else if (responseResult.status == 400) {
+          if (result && result.file && result.file[0] === "This field is required.") {
+            message = `Error ${responseResult.status}: A file is required.`;
+          } else {
+            message = `Error ${responseResult.status}: ${result.file[0]}`;
+          }
+        } else if (responseResult.status == 403) {
+            message = `Error ${responseResult.status}: Je bent niet ingelogd.`;
+        } else {
+          message = `Error ${responseResult.status}: ${responseResult.statusText}`;
+        }
```

Comment:
Looks good.
Might is be a good idea to always log "responseResult.statusText" for unexpected result ?